pkg/cmd/server/origin/ensure.go

```diff
@@ -28,7 +30,7 @@ func (c *MasterConfig) ensureOpenShiftSharedResourcesNamespace() {
 	if _, err := c.KubeClientsetInternal().Core().Namespaces().Get(c.Options.PolicyConfig.OpenShiftSharedResourcesNamespace, metav1.GetOptions{}); kapierror.IsNotFound(err) {
 		namespace, createErr := c.KubeClientsetInternal().Core().Namespaces().Create(&kapi.Namespace{ObjectMeta: metav1.ObjectMeta{Name: c.Options.PolicyConfig.OpenShiftSharedResourcesNamespace}})
 		if createErr != nil {
-			glog.Errorf("Error creating namespace: %v due to %v\n", c.Options.PolicyConfig.OpenShiftSharedResourcesNamespace, createErr)
+			utilruntime.HandleError(fmt.Errorf("error creating namespace: %v due to %v", c.Options.PolicyConfig.OpenShiftSharedResourcesNamespace, createErr))
```

Comment:
1. These changes should be a separate commit.
2. This is completely rewritten by David.